web/settings/modulconfigpv.php

```diff
@@ -277,7 +277,25 @@
 								<div class="col">
 									<input class="form-control" type="number" name="solarview_port" id="solarview_port" value="<?php echo htmlspecialchars($solarview_portold) ?>">
 									<span class="form-text small">
-										Gültige Werte Port, z.B. 15000.
+										Gültige Werte: Port, z.B. 15000
+									</span>
+								</div>
+							</div>
+							<div class="form-row mb-1">
+								<label for="solarview_timeout" class="col-md-4 col-form-label">Timeout für den Solarview TCP-Server</label>
+								<div class="col">
+									<input class="form-control" type="number" name="solarview_timeout" id="solarview_timeout" value="<?php echo htmlspecialchars($solarview_timeoutold) ?>">
+									<span class="form-text small">
+										Gültige Werte: Dauer in Sekunden, z.B. 3
+									</span>
+								</div>
+							</div>
+							<div class="form-row mb-1">
+								<label for="solarview_command_wr" class="col-md-4 col-form-label">Kommando für die Abfrage der Wechselrichter</label>
+								<div class="col">
+								<input class="form-control" type="text" pattern="^\d{2}\*$" name="solarview_command_wr" id="solarview_command_wr" value="<?php echo $solarview_command_wrold ?>">
```

Comment:
Der Parameter `solarview_timeout` muss noch in der `updateConfig.sh` ergänzt werden.
Im Hilfetext sollten die Beispielwerte aus dem Code (`modules/wr_solarview/solarview.py` Zeile 28 ff) für einzelne WR ergänzt werden, da das definitiv nicht selbsterklärend ist.

Reply:
Die Dokumentation ist erweitert.
Zur `updateConfig.sh` warte ich die Antwort auf die vorige Frage ab.

Reply:
Der Parameter `solarview_command_wr` wird jetzt in der `updateConfig.sh` verarbeitet.